tests/suites/hup/tests/smoke.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 balena
+ *
+ * @license Apache-2.0
+ */
+
+'use strict';
+
+module.exports = {
+  title: 'Smoke test',
+  run: async function (test) {
+    await this.context.get().hup.initDUT(
+      this, test, this.context.get().link);
+
+    this.teardown.register(async () => {
```

Comment:
@vipulgupta2048 I decided to pull out the logs unconditionally in this way :) I would probably move it into `initDUT` to avoid forcing all tests setting it up manually like this, wdyt?

Reply:
SGTM